runtime/onert/backend/cpu/ops/OperationUtils.h

```diff
@@ -84,6 +84,9 @@ inline nnfw::cker::Shape getTensorShape(const Tensor *tensor)
   if (tensor == nullptr)
     return nnfw::cker::Shape();
 
+  if (tensor->num_dimensions() == 0)
+    return nnfw::cker::Shape(1, 1);
```

Comment:
@hseok-oh I think this change could affect existing code, that assumes scalar has empty shape (size = 0).

Reply:
This is my mistake (test code). I'll remove this code.